bikesharingweb/pages/helpers.js

```diff
@@ -24,7 +24,7 @@ const helpers = {
         }
 
         // User stored locally isn't valid anymore. Let's clear the local data.
-        clearUserCookie();
+        this.clearUserCookie();
```

Comment:
>this [](start = 8, length = 4)

lol :)

Reply:
Yep... Actually, I'm not sure of how it managed to find the definition afterward.

---
In reply to: [268895445](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/25#discussion_r268895445) [](ancestors = 268895445)